docker-compose.yml

```diff
@@ -83,7 +83,8 @@ services:
       retries: 5
 
   gitlab:
-    image: gitlab/gitlab-ce:latest
+    image: ${DOCKER_PREFIX}gitlab:${PLATFORM_VERSION}
+    build: services/gitlab
```

Comment:
out-of-scope changes (dockerfile), put then in another PR

Reply:
It's related so the environments are created properly for jupyterhub access control checks.

Reply:
the `docker-compose.yml` has ZERO effect on the charts